hypervisor/src/lib.rs

```diff
@@ -49,9 +49,8 @@ pub use crate::hypervisor::{Hypervisor, HypervisorError};
 pub use cpu::{HypervisorCpuError, Vcpu, VmExit};
 pub use device::{Device, HypervisorDeviceError};
 pub use kvm::*;
-pub use vm::{DataMatch, HypervisorVmError, Vm};
-
 use std::sync::Arc;
+pub use vm::{DataMatch, HypervisorVmError, Vm};
```

Comment:
This is just moving code to a different location. It doesn't seem to serve any practical purpose. Please drop this hunk.